modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v10_8_1/ObjectEntryAssetEntryTitleUpgradeProcess.java

```diff
@@ -56,7 +56,6 @@ protected void doUpgrade() throws Exception {
 						"from ObjectDefinition left join ObjectField on ",
 						"ObjectDefinition.titleObjectFieldId = ",
 						"ObjectField.objectFieldId where ",
-						"ObjectDefinition.enableLocalization = [$TRUE$] and ",
```

Comment:
I don't think we need to remove it from this upgrade step. Because, at this point, the column still exists.

Reply:
This will cause a [failure](https://testray.liferay.com/#/project/35392/routines/45357/build/353965693/case-result/353979060)